CHANGELOG.rst

```diff
@@ -23,6 +26,11 @@ Breaking changes:
 * ``Catalog.add_format_validators()`` superseded by ``@format_validator`` / ``Catalog.enable_formats()``
 * Rename ``Catalog.session()`` context manager to ``Catalog.cache()``
 * Rename ``session`` parameter to ``cacheid`` in many places
+* Added ``Catalog.get_metaschema()``, analogous to ``Catalog.get_schema()``
+* ``Catalog.create_metashema()`` and ``Catalog.create_vocabulary()`` return the created instance
+* Rename ``core_vocabulary`` and ``core_vocabulary_uri`` parameters for
+  ``Metaschema.__init__()`` and ``Catalog.create_metaschema()`` respectively to
+  ``default_core_vocabulary`` and ``default_core_vocabulary_uri``
```

Comment:
This can also go under miscellaneous. The params are declared as positional args, so any existing usage of these methods that follow the API and the examples won't need changing.